build.sh

```diff
@@ -448,6 +448,10 @@ BUILD_CUDA="-DUSE_CUDA=OFF"
 BUILD_FPGAOPENCL="-DUSE_FPGAOPENCL=OFF"
 BUILD_DEBUG="-DCMAKE_BUILD_TYPE=Release"
 BUILD_MPI="-DUSE_MPI=OFF"
+BUILD_AVX2="-DUSE_AVX2=OFF"
+BUILD_AVX512="-DUSE_AVX512=OFF"
+BUILD_SCALAR="-DUSE_SCALAR=OFF"
+BUILD_SSE="-DUSE_SSE=OFF"
```

Comment:
an option to completely switch building TSL and SIMD ops on/off similar to CUDA or MPI should be added imho